src/decoder/plugins/GmeDecoderPlugin.cxx

```diff
@@ -105,16 +105,43 @@ ParseContainerPath(Path path_fs)
 	return { path_fs.GetDirectoryName(), track - 1 };
 }
 
-static void
-gme_file_decode(DecoderClient &client, Path path_fs)
-{
-	const auto container = ParseContainerPath(path_fs);
+static Music_Emu
+*LoadGmeAndM3u(GmeContainerPath container) {
+
+	const char *path = container.path.c_str();
+	const char *suffix = uri_get_suffix(path);
 
 	Music_Emu *emu;
 	const char *gme_err =
-		gme_open_file(container.path.c_str(), &emu, GME_SAMPLE_RATE);
+		gme_open_file(path, &emu, GME_SAMPLE_RATE);
 	if (gme_err != nullptr) {
 		LogWarning(gme_domain, gme_err);
+		return nullptr;
+	}
+
+	if(suffix == nullptr) {
+		return emu;
+	}
+
+	std::string m3u_path(path,suffix - path);
+	m3u_path += "m3u";
+
+
+	gme_err = gme_load_m3u(emu,m3u_path.c_str());
+	if(gme_err != nullptr) {
+		LogWarning(gme_domain,gme_err);
```

Comment:
This is not worth a warning. Such a warning in a normal situation (there's no m3u file!) would confuse users.